### PR TITLE
add different types of Card components

### PR DESCRIPTION
**Pull request by Prathyusha-96**
image
image
image
image
image

### CODE REVIEW

**Comment by mohammed-sas** [comment]
kindly add screenshot or csb link while raising pr

**Review by mohammed-sas — CHANGES_REQUESTED** [review]
you could add more spacing for the card body , a better styling for "add to cart" button would be great.

File: components/card/card.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            <div class="card-badge">NEW</div>
+            <button class="ico">
+                <i class="far fa-heart"></i>
+            </button>
+            <div class="card-title">Men's Jacket</div>
+            <div class="card-price">
+                <span class="price-now">Rs.350</span>
+                <span class="price-before">Rs.899</span>
+                <span class="discount">(60% OFF)</span>
+            </div>
```
Comment (Husain01) [inline components/card/card.html:118]: A a bit of margin and padding to give some space.

File: components/card/card.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            <button class="btn btn-primary">ADD TO CART</button>
```
Comment (Husain01) [inline components/card/card.html:119]: Give a bit of padding to the button and cursor: pointer

**Review by Husain01 — CHANGES_REQUESTED** [review]

**Review by mohammed-sas — APPROVED** [review]
suggestion :  add good styling to button in the card

pr good to merge

File: components/card/card.css [hunk 3]
```diff
@@ -156,11 +157,17 @@
     margin-left: 1rem;
 }
 .btn-primary {
```
Comment (Husain01) [inline components/card/card.css:159]: Just a suggestion add border:none or style as per your styling. Everything else's looks good.

**Review by Husain01 — APPROVED** [review]

**Pull request merged**